src/App/Resources/AppResources.resx

```diff
@@ -2275,4 +2275,46 @@
   <data name="AreYouSureYouWantToEnableScreenCapture" xml:space="preserve">
     <value>Are you sure you want to enable Screen Capture?</value>
   </data>
+  <data name="LogInRequested" xml:space="preserve">
+    <value>Log in requested</value>
+  </data>
+  <data name="AreYouTryingToLogIn" xml:space="preserve">
+    <value>Are you trying to log in?</value>
+  </data>
+  <data name="LogInAttemptByOn" xml:space="preserve">
+    <value>Log in attempt by {0} on {1}</value>
+  </data>
```

Comment:
In order be easier to guess, I try to use XYZ letters to indicate that it needs parameters, so I'd change it to:
`LogInAttemptByXOnY`